File: clojure/src/hello_world.clj
(ns hello-world)

(defn print-welcome-message []
  ;; This variable is interpolated by
  ;; Boilermaker at build time, and its value is taken from the `boilermaker.toml` file.
  (let [interpolated-message "{{welcome_message}}"]
    (println interpolated-message)))

(defn -main []
  (println "Hello boiled Clojure!")
  (print-welcome-message))

File: rust/src/hello_world.rs
fn main() {
    println!("Hello boiled Rust!");

    print_welcome_message();
}

pub fn print_welcome_message() {
    // This variable is interpolated by
    // Boilermaker at build time, and its value is taken from the `boilermaker.toml` file.
    let interpolated_message = "{{welcome_message}}";
    println!("{interpolated_message}");
}
